variables.tf

```
@@ -454,3 +454,14 @@ variable "administrator_emails" {
   default     = []
   type        = list(string)
 }
+variable "snapshot_identifier" {
```

This should include 'rds' for clarity

```suggestion
variable "rds_snapshot_identifier" {
```